AFQ/utils/tests/test_path.py

```diff
@@ -0,0 +1,9 @@
+from AFQ.utils.path import drop_extension
+
+
+def test_drop_extension():
+    assert "/my/.example/.path/to_file" == drop_extension(
+        "/my/.example/.path/to_file.nii.gz")
+    
+    assert "C:\Documents\Newsletters\Summer2018" == drop_extension(
```

Comment:
I bet that the current test failures have to do with the fact that the webpage you copied this from was using some funky character set. Might work if you write it out by hand. Oh, Microsoft.

Reply:
I think it was just that I needed to escape the backslash characters!

Reply:
Oh - right. Good call.